### PR TITLE
Fix/updater

### PR DESCRIPTION
**Pull request by bimac**

### CODE REVIEW

File: .gitignore
Comment (oliche) [inline .gitignore:1]: roooo

File: iblrig/__init__.py [hunk 1]
```diff
@@ -1,8 +1,7 @@
-from pathlib import Path
 from setuptools_scm import get_version
```
Comment (oliche) [inline iblrig/__init__.py:1]: Is there an alternative to a new package? In general we're very careful about adding packages that are not part of the standard library or large reputable open source packages.

In this case to get the version number, I would look into alternatives.

**Review by oliche — APPROVED** [review]

File: iblrig/__init__.py (hunk 1 above)
Comment (bimac) [reply iblrig/__init__.py:1]: This one is a bit more specific, as it gets the version numbers from the local .git directory. Alternative would be to go back to hardcoding the version numbers - which wouldn't be a big deal either.

Comment (bimac) [reply iblrig/__init__.py:1]: We would, however, lose support for per-commit versions (e.g., "8.9.3.post3"), but these could probably also be possible with a bit of git magic

Comment (bimac) [reply iblrig/__init__.py:1]: git describe should do the job - I'll look into it

**Pull request merged**